src/main/scala/mesosphere/marathon/core/group/impl/GroupManagerActor.scala

```diff
@@ -133,7 +133,8 @@ private[impl] class GroupManagerActor(
         _ = validateOrThrow(plan)(DeploymentPlan.deploymentPlanValidator(config))
         _ = log.info(s"Computed new deployment plan:\n$plan")
         _ <- scheduler.deploy(plan, force)
-        _ <- groupRepo.storeRoot(plan.target, plan.createdOrUpdatedApps, plan.deletedApps)
+        // TODO(PODS): Add in createdOrUpdatedPods and deletedPods
```

Comment:
should we track this with a separate "pods/" GH issue, or will this be addressed in this PR?